code-challenges.test.js

```diff
@@ -36,25 +52,97 @@ const object3 = { number: -7 }
 
 // b) Create the function that makes the test pass.
 
+const byThree = (num1) => {
+    if (num1.number % 3 === 0) {
+        return `${num1.number} is divisible by three`
+    } else if (num1.number % 3 === 0) {
```

Comment:
This condition and return is the same as on line 56 and 57. You can remove this whole else if entirely.